src/draft/NSG/Agent/LegalEntity/NonListedCompany/SignatoryRights.py

```diff
@@ -0,0 +1,424 @@
+    IDN = "IDN"
+    IRN = "IRN"
+    IRQ = "IRQ"
+    IRL = "IRL"
+    IMN = "IMN"
+    ISR = "ISR"
+    ITA = "ITA"
+    JAM = "JAM"
+    JPN = "JPN"
+    JEY = "JEY"
+    JOR = "JOR"
+    KAZ = "KAZ"
+    KEN = "KEN"
+    KIR = "KIR"
+    PRK = "PRK"
+    KOR = "KOR"
+    KWT = "KWT"
+    KGZ = "KGZ"
+    LAO = "LAO"
+    LVA = "LVA"
+    LBN = "LBN"
+    LSO = "LSO"
+    LBR = "LBR"
+    LBY = "LBY"
+    LIE = "LIE"
+    LTU = "LTU"
+    LUX = "LUX"
+    MAC = "MAC"
+    MKD = "MKD"
+    MDG = "MDG"
+    MWI = "MWI"
+    MYS = "MYS"
+    MDV = "MDV"
+    MLI = "MLI"
+    MLT = "MLT"
+    MHL = "MHL"
+    MTQ = "MTQ"
+    MRT = "MRT"
+    MUS = "MUS"
+    MYT = "MYT"
+    MEX = "MEX"
+    FSM = "FSM"
+    MDA = "MDA"
+    MCO = "MCO"
+    MNG = "MNG"
+    MNE = "MNE"
+    MSR = "MSR"
+    MAR = "MAR"
+    MOZ = "MOZ"
+    MMR = "MMR"
+    NAM = "NAM"
+    NRU = "NRU"
+    NPL = "NPL"
+    NLD = "NLD"
+    ANT = "ANT"
+    NCL = "NCL"
+    NZL = "NZL"
+    NIC = "NIC"
+    NER = "NER"
+    NGA = "NGA"
+    NIU = "NIU"
+    NFK = "NFK"
+    MNP = "MNP"
+    NOR = "NOR"
+    OMN = "OMN"
+    PAK = "PAK"
+    PLW = "PLW"
+    PSE = "PSE"
+    PAN = "PAN"
+    PNG = "PNG"
+    PRY = "PRY"
+    PER = "PER"
+    PHL = "PHL"
+    PCN = "PCN"
+    POL = "POL"
+    PRT = "PRT"
+    PRI = "PRI"
+    QAT = "QAT"
+    REU = "REU"
+    ROU = "ROU"
+    RUS = "RUS"
+    RWA = "RWA"
+    SHN = "SHN"
+    KNA = "KNA"
+    LCA = "LCA"
+    SPM = "SPM"
+    VCT = "VCT"
+    WSM = "WSM"
+    SMR = "SMR"
+    STP = "STP"
+    SAU = "SAU"
+    SEN = "SEN"
+    SRB = "SRB"
+    SYC = "SYC"
+    SLE = "SLE"
+    SGP = "SGP"
+    SVK = "SVK"
+    SVN = "SVN"
+    SLB = "SLB"
+    SOM = "SOM"
+    ZAF = "ZAF"
+    SGS = "SGS"
+    SSD = "SSD"
+    ESP = "ESP"
+    LKA = "LKA"
+    SDN = "SDN"
+    SUR = "SUR"
+    SJM = "SJM"
+    SWZ = "SWZ"
+    SWE = "SWE"
+    CHE = "CHE"
+    SYR = "SYR"
+    TWN = "TWN"
+    TJK = "TJK"
+    TZA = "TZA"
+    THA = "THA"
+    TLS = "TLS"
+    TGO = "TGO"
+    TKL = "TKL"
+    TON = "TON"
+    TTO = "TTO"
+    TUN = "TUN"
+    TUR = "TUR"
+    TKM = "TKM"
+    TCA = "TCA"
+    TUV = "TUV"
+    UGA = "UGA"
+    UKR = "UKR"
+    ARE = "ARE"
+    GBR = "GBR"
+    USA = "USA"
+    UMI = "UMI"
+    URY = "URY"
+    UZB = "UZB"
+    VUT = "VUT"
+    VEN = "VEN"
+    VNM = "VNM"
+    VGB = "VGB"
+    VIR = "VIR"
+    WLF = "WLF"
+    ESH = "ESH"
+    YEM = "YEM"
+    ZMB = "ZMB"
+    ZWE = "ZWE"
+
+
+class Role(str, Enum):
+    DIRECTOR = "director"
+    DEPUTY_DIRECTOR = "deputy director"
+    CHAIRPERSON = "chairperson"
+    BOARD_MEMBER = "board member"
+    DEPUTY_BOARD_MEMBER = "deputy board member"
+    OTHER = "other"
+
+
+class SigningRights(CamelCaseModel):
+    personal_id: Optional[str] = Field(
+        None,
+        title="Personal ID",
+        description="The ID of a person if exists, e.g. social security number or similar",
+        example="1129955131",
+        max_length=40,
+    )
+    role: Role = Field(
```

Comment:
Could we move the role above the personal_id, so the order here would match the order in the spreadsheet?